smartcar/smartcar.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, client_id, client_secret, redirect_uri, scope=None, test_mode
         else:
             self.test_mode = test_mode if test_mode else False
 
-    def get_auth_url(self, force=False, state=None, vehicle_info=None):
+    def get_auth_url(self, force=False, state=None, vehicle_info=None, single_select=False):
```

Comment:
Should we default to `False` though?

I think we should try to be consistent across our SDKs and either decide to add default values to params not passed in or do not default. thoughts?

Mentioning because I ran into the same scenario in Java SDK, so I think it would be a good idea to come to a consensus.

Reply:
Yes I was thinking about this this morning when I was working on iOS I agree on that we should have consistency across SDKs on this! 

The other params currently have mixed behavior if you pass nothing in (some use a default value, others just don't append anything). I think that it's easier to understand if you don't pass anything in for single select, it doesn't show up in the auth url, and same if you pass in incorrect info. This would also match the behavior of our make bypass parameter.